docs/introduction.md

```diff
@@ -1,5 +1,10 @@
 # Introduction to the Magento Functional Testing Framework
 
+<div class="bs-callout bs-callout-info" markdown="1">
+These are the docs for the latest MFTF release.
+To find older documentation, please refer to the `docs` folder inside you desired release in Github.
```

Comment:
Can we add link to the docs folder in Github?

Reply:
Good idea, I'll link to it